geonode/messaging/__init__.py

```diff
@@ -33,6 +34,8 @@
 class MessagingAppConfig(NotificationsAppConfigBase):
     name = 'geonode.messaging'
 
+    NOTIFICATIONS = (("create_message", _("Message received"), _("New message received in one of your threads")),)
```

Comment:
please call this notification "message_recevied"

Reply:
Done